rust/foxglove/src/websocket/ws_protocol/server/server_info.rs

```diff
@@ -91,6 +130,9 @@ pub enum Capability {
     ConnectionGraph,
     /// Allow clients to fetch assets.
     Assets,
+    /// Indicates that the server is sending data within a fixed time range. This requires the
+    /// server to specify the `data_start_time` and `data_end_time` fields in its `ServerInfo` message.
+    RangedPlayback,
```

Comment:
Probably should also mention here that playback control requests are only accepted when the server advertises this capability